integration_tests/models/source/default/snowplow_unified_screen_summary_context_stg.sql

```diff
@@ -0,0 +1,29 @@
+  {#
+Copyright (c) 2023-present Snowplow Analytics Ltd. All rights reserved.
+This program is licensed to you under the Snowplow Personal and Academic License Version 1.0,
+and you may not use this file except in compliance with the Snowplow Personal and Academic License Version 1.0.
+You may obtain a copy of the Snowplow Personal and Academic License Version 1.0 at https://docs.snowplow.io/personal-and-academic-license-1.0/
+#}
+
+Select
+  ev.event_id as root_id,
+  ev.collector_tstamp::timestamp as root_tstamp,
+
+  (ev.contexts_com_snowplowanalytics_mobile_screen_summary_1_0_0::json->0 ->>'foreground_sec')::float as foreground_sec,
+  (ev.contexts_com_snowplowanalytics_mobile_screen_summary_1_0_0::json->0 ->>'background_sec')::float as background_sec,
```

Comment:
is this some fancy json column type i'm too Oracle warehouse to understand?

Reply:
Also should we be doing this for all the other contexts 👀

Reply:
Yeah, that's the syntax for parsing json in Postgres, unfortunately Redshift has slightly different functions 😞 

I think it'd be great to use this for all contexts, would let us remove all those CSV files for the contexts. Probably something for a separate PR though, didn't want to overload this one.